run_tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+conda env create -f requirements.txt
```

Comment:
This will print a single error line in the output if you already have the `bellhop` environment (which I couldn't suppress). Another option would be to force an install each time, which would make running tests take a while each and every time, not great. Note on CircleCI the environment is fresh so no issue there.

Another thought is to have an if statement here with the default behavior as current behavior and then a flag like `-f` for "force fresh install" enabling you to update your conda environment. Thought @thbenschop ?

Reply:
I think most of the scripts should assume already being in the environment, but yeah some script that does a smart setup (checks for env existence, needs updates, need to switch to context) like sort_out_environment.sh (or .py) would be nice

Reply:
that script is in master now